tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
-- name: Include OS-specific variables.
-  include_vars: "{{ ansible_os_family }}.yml"
+- name: set ntp_daemon to 'ntp' as default
```

Comment:
ok ... it is your island and your point of view.
Just started with ansible and was told to keep things simple ... defining a default var (with lowest precedence) is simple and can be overwritten any time later without any problem ... as far as I understand ...
From my point of view it is nonsense to define several times the same var instead of defining _one default_ . I would understand when the var would be different on each distro ... but the var is _same_ on most and hence far away from being _distro-specific_. Only Debian and SLE-11 is different from default ...